examples/go/Dockerfile

```diff
@@ -1,12 +1,21 @@
-FROM golang:1.12 as build-env
+FROM golang:latest as build-env
 
 WORKDIR /go/src/app
-COPY . /go/src/app
+COPY *.go .
 
+RUN go mod init
 RUN go get -d -v ./...
+RUN go vet -v
+RUN go test -v
 
-RUN go build -o /go/bin/app
+# Details on when to utilize cgo: 
+#   https://pkg.go.dev/cmd/cgo  
+RUN CGO_ENABLED=0 go build -o /go/bin/app
+
+# Details on when to use Distroless Base vs Static:
+#   https://github.com/GoogleContainerTools/distroless/blob/main/base/README.md
+#FROM gcr.io/distroless/base
```

Comment:
If we're not going to use this from line, then probably just remove it.

Reply:
removed ... updated PR